Yoast/YoastSEO.js
# Add paper attributes to internal linking research

## Summary

This PR can be summarized in the following changelog entry:

* Uses words from keyphrase, metadescription, title, and subheadings for internal linking suggestions

## Relevant technical choices:

* The current mechanism might generate duplicates of terms used for internal linking if words or phrases used in paper attributes (keyphrase, metadescription, title, and subheadings) also pass the density requirements in the text of the paper itself. This problem will be solved by the same functionality that will take care of different morphological forms (see #2139)

## Test instructions

This PR can be tested by following these steps:

* Use the `example` to check if words from paper attributes are added to the list of relevant words.
* Try adding more tests to check if words from paper attributes end up in the relevant words combinations.

Fixes #2114
